source/photo.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy:Pink</title>
+
+  </head>
+
+  <body>
+    <header class="page-header">
+      <h1 class="visually-hidden">Фотографии пользоватилей</h1>
+      <nav class="main-nav">
+        <a class="logo" href="index.html">
+          <img class="logo-image" src="https://via.placeholder.com/150x150" alt="логотип Pink">
+        </a>
+        <button class="main-nav__toggle" type="button"></button>
+        <ul class="main-nav__list site-list">
+          <li class="site-list__item">
+            <a href="index.html">Главная</a>
+          </li>
+          <li class="site-list__item">
+            <a href="#">Фотографии</a>
+          </li>
+          <li class="site-list__item">
+             <a href="form.html">Конкурс</a>
+          </li>
+          <li class="site-list__item">
+            <a href="https://htmlacademy.ru/intensive/adaptive">HTML Academy</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="page-main">
+      <p class="intro-text">
+        Взгляните на фотографии, которые
+        выкладывают пользователи!
+        Видите, как не хватает ваших?
+      </p>
+      <ul class="userphotos-list">
+        <li class="userphotos-list__item">
+          <span>панорама дня</span>
+          <img src="https://via.placeholder.com/150x150" alt="панорама">
+          <p class="userphotos-list__comment">
+            <span class="userphotos-list__comment-name">Анна</span>
+            <span class="userphotos-list__comment-time">(5 минут назад)</span>
+            Прыгать или нет? Напишите в
+            коментарии свой совет и смотрите
+            прямую трансляцию в перископе,
+            задавайте свои вопросы!
+          </p>
+          <button class="like-button">
```

Comment:
```suggestion
          <button class="like-button" type="button">
```
(у аналогичных кнопок тоже укажите).